tests/generators/OperationGenerator.ts

```diff
@@ -578,18 +576,14 @@ export default class OperationGenerator {
 
   /**
    * Generates a single element array with a identity hub service object for DID document
-   * @param instances the instance field in serviceEndpoint. A list of DIDs
+   * @param id the id field in serviceEndpoint.
    */
-  public static createIdentityHubUserServiceEndpoints (instances: string[]): any[] {
+  public static createIdentityHubUserServiceEndpoints (id: string): any[] {
```

Comment:
This method no longer makes sense in the new context... can we change it so it becomes:
```suggestion
  public static generateServiceEndpoints (count: number): DidServiceEndpointModel[] {
```